module/skill/GetUp/src/GetUp.hpp

```diff
@@ -0,0 +1,36 @@
+#ifndef MODULE_SKILL_GETUP_HPP
+#define MODULE_SKILL_GETUP_HPP
+
+#include <nuclear>
+
+#include "extension/Behaviour.hpp"
+#include "extension/behaviour/Script.hpp"
+
+namespace module::skill {
+
+    class GetUp : public ::extension::behaviour::BehaviourReactor {
+    private:
+        /// @brief Stores configuration values
+        struct Config {
+            /// @brief Script sequence to run when getting from lying on the front to standing
+            std::vector<::extension::behaviour::ScriptRequest> getup_front;
+            /// @brief Script sequence to run when getting from lying on the back to standing
+            std::vector<::extension::behaviour::ScriptRequest> getup_back;
+            /// @brief Script sequence to run when getting from lying on the right to standing
+            std::vector<::extension::behaviour::ScriptRequest> getup_right;
+            /// @brief Script sequence to run when getting from lying on the left to standing
+            std::vector<::extension::behaviour::ScriptRequest> getup_left;
+            /// @brief Script sequence to run when already upright
+            std::vector<::extension::behaviour::ScriptRequest> getup_upright;
+            /// @brief Script sequence to run when told to get up while upside down
```

Comment:
I think upside down is better than inverted. and you should explain why we relax when upside down.